VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/host-hearing-list/staff-member-hearing-list/staff-member-hearing-list.component.ts

```diff
@@ -52,4 +53,22 @@ export class StaffMemberHearingListComponent extends HostHearingListBaseComponen
             })
         );
     }
+
+    onConferenceSelected(conference: ConferenceForHostResponse) {
+        this.logger.debug('[StaffMemberHearingList] - Signing into judge waiting room', { conference: conference.id });
+
+        this.videoWebService.getConferenceById(conference.id).then(conferenceResponse => {
+            this.hearingVenueFlagsService.setHearingVenueIsScottish(conference.hearing_venue_is_scottish);
+            this.profileService.getUserProfile().then(profile => {
+                if (conferenceResponse.participants.some(x => x.user_name === profile.username)) {
+                    this.router.navigate([pageUrls.StaffMemberWaitingRoom, conference.id]);
+                }
+                this.videoWebService
```

Comment:
shouldn't this be in the else block? We only want to call this endpoint if the staff member hasn't been added to the conference yet correct?